### PR TITLE
[SC-100] Remove jumpcloud access

### PR DESCRIPTION
**Pull request by zaro0508**
Only support Connection URI, no more jumpcloud access.

### CODE REVIEW

File: ec2/sc-ec2-windows-jumpcloud.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 AWSTemplateFormatVersion: '2010-09-09'
-Description: 'Service Catalog: Windows EC2 with Jumpcloud integration.'
+Description: 'Service Catalog: Windows EC2'
```
Comment (ahayden) [inline ec2/sc-ec2-windows-jumpcloud.yaml:2]: We actually need to keep Jumpcloud on the Windows machines. Please leave the connector on this product.

Comment (zaro0508) [reply ec2/sc-ec2-windows-jumpcloud.yaml]: done

File: ec2/sc-ec2-windows-jumpcloud.yaml [hunk 2]
```diff
@@ -94,7 +91,7 @@ Resources:
                  [cfn-auto-reloader-hook]
                  triggers=post.update
                  path=Resources.WindowsInstance.Metadata.AWS::CloudFormation::Init
-                 action=cfn-init.exe -v --stack ${AWS::StackId} --resource WindowsInstance --region ${AWS::Region} --configsets SetupCfn,SetupApps,SetEnv,SetTags,SetupJumpcloud
+                 action=cfn-init.exe -v --stack ${AWS::StackId} --resource WindowsInstance --region ${AWS::Region} --configsets SetupCfn,SetupApps,SetEnv,SetTags
```
Comment (ahayden) [inline ec2/sc-ec2-windows-jumpcloud.yaml:94]: You'll need to leave the SetupJumpcloud directive in the cfn-init hooks, too.

File: ec2/sc-ec2-windows-jumpcloud.yaml [hunk 3]
```diff
@@ -226,7 +223,7 @@ Resources:
       UserData:
         Fn::Base64: !Sub |
           <script>
-          cfn-init.exe -v --stack ${AWS::StackId} --resource WindowsInstance --region ${AWS::Region} --configsets SetupCfn,SetupApps,SetEnv,SetTags,SetupJumpcloud
+          cfn-init.exe -v --stack ${AWS::StackId} --resource WindowsInstance --region ${AWS::Region} --configsets SetupCfn,SetupApps,SetEnv,SetTags
```
Comment (ahayden) [inline ec2/sc-ec2-windows-jumpcloud.yaml:226]: The cfn-init hook for SetupJumpcloud needs to stay here too

Comment (zaro0508) [reply ec2/sc-ec2-windows-jumpcloud.yaml]: done

**Pull request merged**